android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -31,6 +31,10 @@ function init() {
   keyman.getOskHeight = getOskHeight;
   keyman.getOskWidth = getOskWidth;
   keyman.beepKeyboard = beepKeyboard;
+
+  // Readies the keyboard stub for instant loading during the init process.
+  KeymanWeb.registerStub(JSON.parse(jsInterface.initialKeyboard()));
```

Comment:
Uses the function defined within this PR within the Java `KMKeyboardJSHandler` class.  See other comment (re https://developer.apple.com/documentation/webkit/wkscriptmessagehandlerwithreply) on how this _could_ look for iOS if we wanted to go that route.

That said, I don't see the same problem on my personal iOS device, so it can probably be low-priority for now.